.gitattributes

```diff
@@ -0,0 +1,2 @@
+/.idea export-ignore
```

Comment:
Updated, also with a view to GrumPHPAlso with a view to GrumPHP (https://github.com/bunq/sdk_php/pull/24/).